### DISCUSSION TITLE
How to consume a NuGet package in **dotnet/corefx** projects?

### DISCUSSION THREAD

**Discussion by DavoudEshtehari** [dotnet/runtime#66603] [Q&A]
My assumption is, I could consume a NuGet which is defined in the [public feeds](https://github.com/dotnet/corefx/blob/release/3.1/NuGet.config), using the `PackageReference ` tag in [dotnet/corefx](https://github.com/dotnet/corefx/blob/release/3.1) projects. But the project configurations don't allow it except on test projects.

My target is [System.Data.SqlClient](https://github.com/dotnet/corefx/tree/release/3.1/src/System.Data.SqlClient) project with package referencing [Microsoft.SqlServer.Server](https://www.nuget.org/packages/Microsoft.sqlserver.server). How can I do it?

- Related issue #66531

**Comment by huoyaoyuan** [comment]
Isn't it just on nuget.org?

**Reply by DavoudEshtehari** [reply]
It's accessible either on `nuget.org` or `dotnet-public` feeds.

**Reply by huoyaoyuan** [reply]
The feeds are for daily builds. The versions on `nuget.org` are considered released.

**Reply by DavoudEshtehari** [reply]
Thank you for your response but I don't think it could be the answer to my question.
